wrangler.toml.template

```diff
@@ -17,3 +17,5 @@ command = ""
 
 [vars]
 CODA_TOKEN = "{CODA_TOKEN}"
+CODA_INCOMING_TOKEN = "{CODA_INCOMING_TOKEN}"
+NLP_SEARCH_ENDPOINT = "https://stampy-nlp.uw.r.appspot.com"
```

Comment:
I thought it better to configure urls etc. that can change via wrangler, rather than having them directly in the code - should I change it back?